_topic_maps/_topic_map.yml

```diff
@@ -2966,6 +2966,8 @@ Topics:
     File: ztp-sno-additional-worker-node
   - Name: Pre-caching images for single-node OpenShift deployments
     File: ztp-precaching-tool
+  - Name: Image-based installation and upgrade
```

Comment:
Isn't this just upgrades? 

What about Image-based upgrades for single-node OpenShift clusters